source/endpoints/devices/__init__.py

```diff
@@ -0,0 +1,7 @@
+from flask_restplus import Namespace
+
+from model.Database import Database
+
+DAO = Database()
+
+ns = Namespace('devices', description='All devices of the system')
```

Comment:
No need to call it something as short as 'ns', it only forces you to find this file and see what it _actually_ stands for.